.envrc

```diff
@@ -18,6 +18,9 @@ exports () {
 
     # Add executables to path
     export PATH=$DIR/bin:$PATH
+
+    # Base path for looking up the Apalache standard library
+    export APALACHE_HOME=$DIR
```

Comment:
Should we choose a name that more explicitly says that this is a dev-only variable?
Something like `APALACHE_INTERNAL_HOME` or `APALACHE_DEV_HOME`?

FWIW, even if a user sets this, nothing bad will happen, as we only use it when not running from the packaged JAR.